### DISCUSSION TITLE
What is the correct format for S3_ENDPOINT_URL?

### DISCUSSION THREAD

**Discussion by cbc02009** [TandoorRecipes/recipes#2417] [Q&A]
I have a minio instance at `http://minio.<domain>:9001`. What should I set for `S3_ENDPOINT_URL`? When I use that value I get an error saying tcp is not a valid port number.

**Comment by vabene1111** [comment]
This should hopefully explain everything, the variables names of tandoor are pretty close to the parameters described there

https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html
